pyproject.toml

```diff
@@ -1,3 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2021-2022 The Foundry Visionmongers Ltd
+
+[build-system]
+requires = [
+    "setuptools==65.5.0",
```

Comment:
I would personally just put a minimum version instead of pinning completely that dependency, `setuptools>=65.5.0`.

Reply:
Good shout, updated to use a minimum version range in d1496af5